hs_core/management/commands/remove_unreferenced_files.py

```diff
@@ -0,0 +1,54 @@
+from django.core.management.base import BaseCommand
+from django_irods.storage import IrodsStorage
+from hs_core.models import BaseResource, ResourceFile
+from django.utils import timezone
+from django_irods.icommands import SessionException
+from datetime import timedelta
+
+
+class Command(BaseCommand):
+    help = "Check synchronization between iRODS and Django."
+
+    def add_arguments(self, parser):
+        parser.add_argument('resource_ids', nargs='*', type=str)
+        parser.add_argument('--updated_since', type=int, dest='updated_since',
+                            help='include only resources updated in the last X days')
+        
+    def handle(self, *args, **options):
+        resources_ids = options['resource_ids']
+        updated_since = options['updated_since']
+        resources = BaseResource.objects
+
+        if resources_ids:  # an array of resource short_id to check.
+            print("Setting isPublic AVU for the resources provided")
+            resources = resources.filter(short_id__in=resources_ids)
```

Comment:
maybe you included the isPublic AVU here on accident?

Reply:
yep, copy pasta